src/foam/u2/filter/FilterView.js

```diff
@@ -311,7 +310,7 @@ foam.CLASS({
 
           var e = this.E();
           var labelSlot = foam.core.ExpressionSlot.create({ args: [this.filterController.activeFilterCount$],
-            code: function(x) { return `Filters (${x})`; }});
+            code: function(x) { if ( x > 0 ) return `Filters (${x})`; return `Filters` }});
```

Comment:
this could just be  function (x) { return x > 0 ?  `Filters (${x})` : `Filters`; }

Also Filters needs to use the message not the actual text value

Reply:
ok